packs/opsgenie/actions/add_heartbeat.py

```diff
@@ -0,0 +1,40 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+from lib.actions import OpsGenieBaseAction
+
+
+class AddHeartbeatAction(OpsGenieBaseAction):
+    def run(self, name, interval=None, intervalUnit=None, description=None, enabled=False):
+        """
```

Comment:
No worries, I'm working my way though adding docstrings and they should all be covered in he next commit.